src/file.js

```diff
@@ -1048,9 +1065,7 @@ File.prototype.download = function(options, callback) {
       .on('error', callback)
       .on('finish', callback);
   } else {
-    fileStream
-      .on('error', callback)
-      .pipe(concat(callback.bind(null, null)));
+    fileStream.on('error', callback).pipe(concat(callback.bind(null, null)));
```

Comment:
This is less readable to me than the typical notation where new lines help show the directions a stream goes and which stream gets the event handlers.